### ISSUE TITLE
Error: new BigNumber() not a base 16 number:

### ISSUE THREAD

**Issue by gnurtcouq** [IBM/loyalty-points-evm-fabric#11]
At step 5. Run the web application, after registering a Partner, when go to Sign In page, there is error

Error: new BigNumber() not a base 16 number:

In console, it shows:

Getting the Contract
Got address: 30a5c90d8593a94a89fdcb7a5e2aeb065f18b7af
Getting the Contract
{ BigNumber Error: new BigNumber() not a base 16 number: 
    at raise (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/bignumber.js/bignumber.js:1177:25)
    at /home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/bignumber.js/bignumber.js:1165:33
    at new BigNumber (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/bignumber.js/bignumber.js:212:28)
    at SolidityTypeString.formatOutputString [as _outputFormatter] (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/web3/lib/solidity/formatters.js:220:19)
    at /home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/web3/lib/solidity/type.js:246:25
    at SolidityTypeString.SolidityType.decode (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/web3/lib/solidity/type.js:247:11)
....

Any ideal how to fix this?

**Comment by gnurtcouq** [comment]
Hi All,

Finally I have successfully register a Partner and a Member into the Network. I managed to login as the new Member, then select a Partner using drop-down list . When I click the button "Purchase Item X for $20 and earn 30 points", there is error in the console:

> node app.js

app running on port: 8000
Getting the Contract
Got address: 0d2dc1658066c63ab46d2f772b7e2074766d4350
Getting the Contract
AccountCode: 0xd5c456b7218e9d11c577bd015c61aee93458157f
memberData:0xd5c456b7218e9d11c577bd015c61aee93458157f,Hong,Le,le@yahoo.com,0,true
Getting the Contract
Got address: d9da3fc47ff1d292d1eba3338ab68636bf20a6e7
Getting the Contract
Got address: d9da3fc47ff1d292d1eba3338ab68636bf20a6e7
Using param - points: 30 partnerAddress: 0x64602e1922b94c026785cdd5ea729822bc1207a4
Getting the Contract
Got address: d9da3fc47ff1d292d1eba3338ab68636bf20a6e7
**Error: Failed to execute transaction: Transaction processing for endorser [localhost:7051]: Chaincode status Code: (500) UNKNOWN. Description: failed to execute contract: call error: REVERT not yet fully implemented**
    at Object.InvalidResponse (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/web3/lib/web3/errors.js:38:16)
    at RequestManager.send (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/web3/lib/web3/requestmanager.js:61:22)
    at Eth.send [as sendTransaction] (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/web3/lib/web3/method.js:145:58)
    at SolidityFunction.sendTransaction (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/web3/lib/web3/function.js:170:26)
    at SolidityFunction.execute (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/web3/lib/web3/function.js:256:37)
    at Object.earnPoints (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/dapp.js:627:33)
    at /home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/app.js:180:25
    at Layer.handle [as handle_request] (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/trung/go/src/github.com/hyperledger/loyalty-points-evm-fabric/web-app/node_modules/express/lib/router/route.js:112:3)

**Comment by horeaporutiu** [comment]
@gnurtcouq the `{ BigNumber Error: new BigNumber() not a base 16 number:` error usually occurs if you try to redeem points without picking a partner first. 

I have not seen this problem with `Error: Failed to execute transaction: Transaction processing for endorser [localhost:7051]: Chaincode status Code: (500) UNKNOWN. Description: failed to execute contract: call error: REVERT not yet fully implemented`

.. are you still running into that chaincode status problem? When I have ran the code pattern earlier today, I did not run into that problem.
